scripts/mkdist.js

```diff
@@ -14,43 +14,40 @@ function mkdist (opts, cb) {
 
   var dist = opts.dist
   var ver = dist.version
-  if (ver.match(/^v/)) ver = ver.substring(1)
```

Comment:
Yep. For now, though, that's how me and @haadcode agreed to do it. We can change this later.